backend/.env.template

```diff
@@ -56,5 +56,10 @@ WEBHOOK_ELOPAGE_SECRET=$WEBHOOK_ELOPAGE_SECRET
 # EventProtocol
 EVENT_PROTOCOL_DISABLED=$EVENT_PROTOCOL_DISABLED
 
-# DHT
+# DHT/Federation
 DHT_TOPIC=$DHT_TOPIC
+FEDERATE_COMMUNITY_NAME=$FEDERATE_COMMUNITY_NAME
+FEDERATE_COMMUNITY_URL=$FEDERATE_COMMUNITY_URL
+FEDERATE_COMMUNITY_PORT=$FEDERATE_COMMUNITY_PORT
+FEDERATE_COMMUNITY_UUID=$FEDERATE_COMMUNITY_UUID
+FEDERATE_KEY_SECRET=$FEDERATE_KEY_SECRET
```

Comment:
you use none of this yet.
Consider to only introduce the secret/seed, nothing else to limit the scope of the PR. Changing the config again is cheap and a well defined process.

If you need the work you done in this PR for followup work, just merge this branch into your next to keep working on that state.

Reply:
- Correct in this PR none of them is used,
- the next PR #2339 uses them and for working on, i merged the previous PR in branch of issue #1887
- the third PR #2372 uses then all changes from the previous two PRs and i merged both in the branch of issue #2368 for working on
- the problem will be during working on the 3rd PR to seperate changings belonging to the previous PRs. That means as soon as such uncritical changes can be merged in the master, as easier will be the separation efforts. And the amount of changes can be keept smaller as you all mentioned about my other big PRs ;-)

Reply:
We will not merge this PR, this is off scope and implements parts which are not used. Please stick to the negotiated scope: I even made a ticket to specify it correctly -> https://github.com/gradido/gradido/issues/2375

Reply:
- I followed your requirements to create small PRs with less changed files.
- To go further the next steps will be done on PR #2339 and have to merge the changes from this PR
- To work on the the 3rd step #2372 a merge of all changes from the previous PRs is necessary
- Now my question:
  *  If the previous PRs will not be merged, how can then the amount of changes for the following PRs be kept low?
  * I do not understand what the ticket #2375 will correct against this PR?

Reply:
Sorry i recognize, that i wrote this comment, but doesn't save it... ;-)

- Correct in this PR none of them is used,
- the next PR #2339 uses them and for working on, i merged the previous PR in branch of issue #1887
- the third PR #2372 uses then all changes from the previous two PRs and i merged both in the branch of issue #2368 for working on
- the problem will be during working on the 3rd PR to seperate changings belonging to the previous PRs. That means as soon as such uncritical changes can be merged in the master, as easier will be the separation efforts. And the amount of changes can be keept smaller as you all mentioned about my other big PRs ;-)